src/worker/sandbox/sock.go

```diff
@@ -3,14 +3,14 @@ package sandbox
 import (
```

Comment:
Let's just iterate on cgroups until we're happy with how we want to do logging.  Other subsystems (like SOCK) should be other pull requests.